orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -1,13 +1,14 @@
-from PyQt4 import QtGui
+from itertools import chain
+
 from PyQt4 import QtCore
-from PyQt4.QtGui import *
+from PyQt4 import QtGui
```

Comment:
`QtGui` is unused? Why were imports rearranged? There's always [`git add -p`](http://johnkary.net/blog/git-add-p-the-most-powerful-git-feature-youre-not-using-yet/).